ocfweb/docs/docs/staff/backend/backups.md

```diff
@@ -24,6 +24,10 @@ with only OCF officers to make sure the backups are kept as secure as possible,
 since they contain all of the OCF's important data.  The backups are already
 encrypted, but it doesn't hurt to add a little extra security to that.
 
+### Retention
+
+Off-site backups older than six months (180 days) are permanently deleted.
```

Comment:
I would recommend mentioning and linking to the `prune-old-backups` script as well